rust/driver_manager/src/lib.rs

```diff
@@ -1753,6 +1753,18 @@ fn user_config_dir() -> Option<PathBuf> {
     }
 }
 
+fn system_config_dir() -> Option<PathBuf> {
+    #[cfg(target_os = "macos")]
+    {
+        Some(PathBuf::from("/Library/Application Support/ADBC"))
+    }
+
+    #[cfg(all(unix, not(target_os = "macos")))]
+    {
+        Some(PathBuf::from("/etc/adbc"))
+    }
```

Comment:
Do we need a `#[cfg]` to catch anything remaining and return `None`?

Reply:
I think you're right. I didn't test but this would be a compilation error on targets like Windows and WASM right? I'll fix it.

Reply:
Changed in ee11c0137. I think that will do it?